src/rororo/openapi/core_data.py

```diff
@@ -36,11 +36,13 @@ def find_core_operation(
         return None
 
 
-def get_core_operation(spec: Spec, operation_id: str) -> Operation:
-    for path in spec.paths.values():
-        for operation in path.operations.values():
-            if operation.operation_id == operation_id:
-                return operation
+def get_core_operation(
```

Comment:
it can be implemented using SpecPath features
```python
def get_core_operation(
    spec: SpecPath, operation_id: str
) -> Tuple[str, str]:
    for path_name, path_spec in (spec / "paths").items():
        for http_method, oper_spec in path_spec.items():
            if (oper_spec / "operationId").content() == operation_id:
                    return http_method, path_name
```